next.config.js

```diff
@@ -23,6 +23,11 @@ const moduleExports = {
   images: {
     loader: 'default',
   },
+
+  env: {
+    ANALYTICS_ENABLED: process.env.ANALYTICS_ENABLED == 'true' || 'false',
```

Comment:
This doesn't seem right type-wise.

The
```ts
process.env.ANALYTICS_ENABLED == 'true'
```
evaluates to `true` as in boolean, but the `||` condition returns a string `'false'`.
What data type should it be?

Reply:
It was a string, but I agree it's best as a boolean
https://github.com/LBHackney-IT/housing-repairs-online-frontend/pull/82/commits/2051e2ff9e2b2675188e8e2a443752b69446f09c